runtime/resolvers/metricsview_time_range.go

```diff
@@ -110,54 +102,22 @@ func (r *metricsViewTimeRangeResolver) Validate(ctx context.Context) error {
 }
 
 func (r *metricsViewTimeRangeResolver) ResolveInteractive(ctx context.Context) (runtime.ResolverResult, error) {
-	olap, release, err := r.runtime.OLAP(ctx, r.instanceID, r.mv.Connector)
-	if err != nil {
-		return nil, err
-	}
-	defer release()
-
-	var tr *runtimev1.TimeRangeSummary
-	switch olap.Dialect() {
-	case drivers.DialectDuckDB:
-		tr, err = r.resolveDuckDB(ctx, olap, r.mv.TimeDimension, escapeMetricsViewTable(drivers.DialectDuckDB, r.mv), r.resolvedMVSecurity.RowFilter(), r.args.Priority)
-	case drivers.DialectDruid:
-		tr, err = r.resolveDruid(ctx, olap, r.mv.TimeDimension, escapeMetricsViewTable(drivers.DialectDruid, r.mv), r.resolvedMVSecurity.RowFilter(), r.args.Priority)
-	case drivers.DialectClickHouse:
-		tr, err = r.resolveClickHouseAndPinot(ctx, olap, r.mv.TimeDimension, escapeMetricsViewTable(drivers.DialectClickHouse, r.mv), r.resolvedMVSecurity.RowFilter(), r.args.Priority)
-	case drivers.DialectPinot:
-		tr, err = r.resolveClickHouseAndPinot(ctx, olap, r.mv.TimeDimension, escapeMetricsViewTable(drivers.DialectPinot, r.mv), r.resolvedMVSecurity.RowFilter(), r.args.Priority)
-	default:
-		return nil, fmt.Errorf("not available for dialect '%s'", olap.Dialect())
-	}
+	ts, err := r.executor.Timestamps(ctx, nil)
 	if err != nil {
 		return nil, err
 	}
 
 	row := map[string]any{}
-	if tr.Min != nil {
-		row["min"] = tr.Min.AsTime()
-		row["max"] = tr.Max.AsTime()
-		row["interval"] = map[string]any{
-			"days":   tr.Interval.Days,
-			"months": tr.Interval.Months,
-			"micros": tr.Interval.Micros,
-		}
+	if !ts.Min.IsZero() {
+		row["min"] = ts.Min
+		row["max"] = ts.Max
+		row["watermark"] = ts.Watermark
```

Comment:
Actually it would be best if this always has the same map keys – otherwise the `schema` will not match the `row` values. So if a timestamp is null, it should be added as `row["key"] = nil`.